<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GTAActionFailureWidget</class>
 <widget class="QWidget" name="GTAActionFailureWidget">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>538</width>
    <height>558</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Failure Command</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="margin">
       <number>2</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QComboBox" name="failureTypeCB">
          <item>
           <property name="text">
            <string>Start</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Stop</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Failure</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Resume</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="searchFailureChB">
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Search Existing Failure</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Failure Name       </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="failureNameLE"/>
        </item>
        <item>
         <widget class="QToolButton" name="searchFailurePB">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="../resource.qrc">
            <normaloff>:/images/forms/img/SEARCH.png</normaloff>:/images/forms/img/SEARCH.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QFrame" name="stopItemFrame">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="margin">
       <number>2</number>
      </property>
      <property name="spacing">
       <number>2</number>
      </property>
      <item row="1" column="0" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string> Parameter Value</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="parameterValLE">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="setConfigValTB">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../resource.qrc">
            <normaloff>:/images/forms/img/failure.png</normaloff>:/images/forms/img/failure.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <widget class="QListWidget" name="paramValLW"/>
      </item>
      <item row="2" column="1">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QToolButton" name="editPB">
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="../resource.qrc">
            <normaloff>:/images/forms/img/EDIT.png</normaloff>:/images/forms/img/EDIT.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="addPB">
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="../resource.qrc">
            <normaloff>:/images/forms/img/Add.png</normaloff>:/images/forms/img/Add.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="deletePB">
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="../resource.qrc">
            <normaloff>:/images/forms/img/Remove.png</normaloff>:/images/forms/img/Remove.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="0" column="0" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string> Parameter Name</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="parameterNameLE"/>
        </item>
        <item>
         <widget class="QToolButton" name="searchPB">
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="../resource.qrc">
            <normaloff>:/images/forms/img/SEARCH.png</normaloff>:/images/forms/img/SEARCH.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Maximum</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resource.qrc"/>
 </resources>
 <connections/>
</ui>
